pkg/daemon/ceph/client/filesystem_mirror.go

```diff
@@ -212,8 +213,16 @@ func GetFSMirrorDaemonStatus(context *clusterd.Context, clusterInfo *ClusterInfo
 	// Using Debug level since this is called in a recurrent go routine
 	logger.Debugf("retrieving filesystem mirror status for filesystem %q", fsName)
 
+	// If cephVersion >= 16.2.6, we run `ceph fs snapshot mirror daemon status`
+	// otherwise we run `ceph fs snapshot mirror daemon status <fs_name>`
+	cephVersion := clusterInfo.CephVersion.IsAtLeast(cephver.CephVersion{Major: 16, Minor: 2, Extra: 6})
```

Comment:
A unit test is needed to check the behavior for different versions

Reply:
Done